### PR TITLE
[4.0] Installation: Fix arrow to be visible in dropdown

### PR DESCRIPTION
**Pull request by Quy**
Pull Request for Issue #28050 .

### Summary of Changes
Arrow in dropdown is not visible during installation.
Replace light arrow with dark arrow svg files.

### Testing Instructions
Apply PR.
Delete configuration.php.
Do new install.
See visible arrow in dropdown.

### CODE REVIEW

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 9b64f45ff331a258a972bde253f1727bb20a0a37<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28059">issues.joomla.org/tracker/joomla-cms/28059</a>.</sub>

File: installation/template/images/select-bg.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1854.54 295" preserveAspectRatio="xMinYMid" width="1854.54" height="295"><polyline points="1825.08 144.47 1832.14 151.53 1839.01 144.66" fill="none" stroke="#fff" stroke-miterlimit="10" stroke-width="1"/><rect width="1810" height="295" fill="#fefefe"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1854.54 295" preserveAspectRatio="xMinYMid" width="1854.54" height="295"><path d="M1825.1,145.7l6.9,6.9c0.1,0.1,0.2,0.1,0.3,0.1c0.1,0,0.2,0,0.3-0.1l6.9-6.9c0.1-0.1,0.1-0.2,0.1-0.3c0-0.1,0-0.2-0.1-0.3l-0.7-0.7c-0.1-0.1-0.2-0.1-0.3-0.1s-0.2,0-0.3,0.1l-5.8,5.8l-5.8-5.8c-0.1-0.1-0.2-0.1-0.3-0.1c-0.1,0-0.2,0-0.3,0.1l-0.7,0.7c-0.1,0.1-0.1,0.2-0.1,0.3C1824.9,145.5,1825,145.6,1825.1,145.7z" fill="#000"/><rect width="1810" height="295" fill="#fff"/></svg>
```
Comment (C-Lodder) [inline installation/template/images/select-bg.svg:1]: SVG seems rather big for just a chevon. Try this:

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 256"><path fill="#000" d="M225.813 48.907L128 146.72 30.187 48.907 0 79.093l128 128 128-128z"/></svg>
```

File: installation/template/images/select-bg-rtl.svg [hunk 2]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1854.54 295" transform="scale(-1,1)" preserveAspectRatio="xMinYMid" width="1854.54" height="295"><path d="M1825.1,145.7l6.9,6.9c0.1,0.1,0.2,0.1,0.3,0.1c0.1,0,0.2,0,0.3-0.1l6.9-6.9c0.1-0.1,0.1-0.2,0.1-0.3c0-0.1,0-0.2-0.1-0.3l-0.7-0.7c-0.1-0.1-0.2-0.1-0.3-0.1s-0.2,0-0.3,0.1l-5.8,5.8l-5.8-5.8c-0.1-0.1-0.2-0.1-0.3-0.1c-0.1,0-0.2,0-0.3,0.1l-0.7,0.7c-0.1,0.1-0.1,0.2-0.1,0.3C1824.9,145.5,1825,145.6,1825.1,145.7z" fill="#fff"/><rect width="1810" height="295" fill="#fff"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="1854.539" height="295" preserveAspectRatio="xMinYMin meet"><path d="M13.573 145.7l6.9 6.9c.1.1.2.1.3.1s.2 0 .3-.1l6.9-6.9c.1-.1.1-.2.1-.3s0-.2-.1-.3l-.7-.7c-.1-.1-.2-.1-.3-.1s-.2 0-.3.1l-5.8 5.8-5.8-5.8c-.1-.1-.2-.1-.3-.1s-.2 0-.3.1l-.7.7c-.1.1-.1.2-.1.3-.296.1-.195.2-.096.3h-.002z"/><path fill="#fff" d="M44.54 0h1810v295h-1810z"/></svg>
```
Comment (C-Lodder) [inline installation/template/images/select-bg-rtl.svg:1]: SVG doesn't look correct at all. Change the `fill` to `#000` and open in your browser. See my other comment for a simpler approach

**Comment by Quy** [comment]
These files were copied from `/administrator/templates/atum/images/`. 

With your suggestion, there is no arrow.
arrow-svg

**Comment by C-Lodder** [comment]
@Quy Try setting the `width/height` too.

**Comment by Quy** [comment]
I did the width/height, but no go. Please provide the exact code to use. Thanks.

**Comment by infograf768** [comment]
Replacing both svg (rtl and ltr) in installation with the ones from Atum works perfectly.

I see no reason to create different svg. + if in the future, for any reason we need to modify these again, it will be simpler.

Screen Shot 2020-02-25 at 08 20 42

Screen Shot 2020-02-25 at 08 21 33

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 9b64f45ff331a258a972bde253f1727bb20a0a37<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28059">issues.joomla.org/tracker/joomla-cms/28059</a>.</sub>

**Comment by infograf768** [comment]
rtc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28059">issues.joomla.org/tracker/joomla-cms/28059</a>.</sub>

**Comment by richard67** [comment]
> Replacing both svg (rtl and ltr) in installation with the ones from Atum works perfectly.
> 
> I see no reason to create different svg. + if in the future, for any reason we need to modify these again, it will be simpler.

Agree.

**Comment by richard67** [comment]
Merging as it is an improvement to the current state.

@C-Lodder Feel free to make a PR to improve the SVGs further if necessary/useful.

**Comment by richard67** [comment]
Thanks.

**Pull request merged**